src/main/java/seedu/address/model/person/exceptions/DuplicatePolicyException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.person.exceptions;
```

Comment:
Perhaps this should be in policy.exceptions?

Reply:
Yes. Will change this.